client/src/modules/patients/record/bh-find-document.component.js

```diff
@@ -104,10 +104,10 @@ function FindDocumentComponent(Patient, Modal, Document, Notify) {
     let ext;
 
     if (mimetype.indexOf('image') > -1) {
-      ext =
-        (mimetype.indexOf('jpg') > -1 || mimetype.indexOf('jpeg') > -1) ? '.jpg' :
-          (mimetype.indexOf('png') > -1) ? '.png' :
-            (mimetype.indexOf('gif') > -1) ? '.gif' : '';
+      /* eslint-disable no-nested-ternary */
+      ext = (mimetype.indexOf('jpg') > -1 || mimetype.indexOf('jpeg') > -1) ? '.jpg'
```

Comment:
We should probably make this into a `MIMETypeService` that performs this operation.  That way, we can test it easier.